### PR TITLE
Fix Android 4.4 crash on file move/copy, closes #1239

### PR DESCRIPTION
**Pull request by harshad1**
Potential fix for #1329

### CODE REVIEW

File: app/thirdparty/java/other/writeily/model/WrMarkorSingleton.java [hunk 1]
```diff
@@ -12,17 +12,13 @@
...
 import android.content.Context;
 import android.support.v4.provider.DocumentFile;
-import android.widget.Toast;
 
 import net.gsantner.markor.format.TextFormat;
 import net.gsantner.markor.ui.SearchOrCustomTextDialogCreator;
 import net.gsantner.markor.util.ShareUtil;
-
-import org.apache.commons.io.IOUtils;
+import net.gsantner.opoc.util.FileUtils;
```
Comment (harshad1) [inline app/thirdparty/java/other/writeily/model/WrMarkorSingleton.java:19]: @gsantner Is using opoc here ok?

Comment (gsantner) [reply app/thirdparty/java/other/writeily/model/WrMarkorSingleton.java:19]: Sure. No problem opoc can be used anywhere

**Comment by harshad1** [comment]
This is good to go once @whunger confirms it is working

**Comment by harshad1** [comment]
@gsantner this is ready to go

**Pull request merged**